telemetry_api/lib/telemetry_api_web/controllers/operator_controller.ex

```diff
@@ -2,32 +2,37 @@ defmodule TelemetryApiWeb.OperatorController do
   use TelemetryApiWeb, :controller
 
   alias TelemetryApi.Operators
+  alias TelemetryApi.Utils
   alias TelemetryApi.Operators.Operator
 
   action_fallback TelemetryApiWeb.FallbackController
 
-  defp return_error(conn, message) do
-    conn
-    |> put_status(:bad_request)
-    |> put_resp_content_type("application/json")
-    |> send_resp(:bad_request, Jason.encode!(%{error: message}))
-  end
+  @create_params [
+    "version",
+    "signature",
+    "eth_rpc_url",
+    "eth_rpc_url_fallback",
+    "eth_ws_url",
+    "eth_ws_url_fallback"
+  ]
```

Comment:
There should be a different way to handle this, but maybe it's something for a different PR

Reply:
Done